chef_postgres/recipes/base_backup.rb

```diff
@@ -26,14 +26,13 @@
                         standby: node["chef_postgres"]["pg_config"]["cluster_type"] } })
 end
 
+
 bash "create_base_backup" do
-  code <<-EOF_CBB
-  rm -rf #{backup_dir}/base_backup/*
-  pg_basebackup -d 'host=localhost user=#{repl_user} password=#{repl_pass}' -D #{backup_dir}/base_backup --xlog-method=stream
-  tar -C #{backup_dir} -czf #{backup_dir}/base_backup.tgz #{backup_dir}/base_backup/  
-  EOF_CBB
-  action :run
-  notifies :run, "ruby_block[log_create_base_backup]", :before
+    code <<-EOF_CBB
+    rm -rf #{backup_dir}/base_backup/*
+    pg_basebackup -d 'host=localhost user=#{repl_user} password=#{repl_pass}' -D #{backup_dir}/base_backup
```

Comment:
We're still missing `--wal-method=stream`

Reply:
this was because we have a base_backup.rb and we have a base backup in the master.rb.  I updated this one as well.